packages/editor/src/Editor.stories.tsx

```diff
@@ -2,37 +2,137 @@ import { storiesOf } from "@storybook/react";
 import * as React from "react";
 import { Value } from "slate";
 import { CivilEditor } from "./Editor";
-import { Renderer } from "./Renderer";
+import { plugins } from "./plugins";
+import { paragraphJson } from "./storyFixtures/paragraph";
+import { headerJson } from "./storyFixtures/header";
+import { ulJson } from "./storyFixtures/ul";
+import { olJson } from "./storyFixtures/ol";
+import { blockquoteJson } from "./storyFixtures/blockquote";
+import { dropCapJson } from "./storyFixtures/dropCap";
+import { boldJson } from "./storyFixtures/bold";
+import { italicJson } from "./storyFixtures/italic";
+import { strikeThroughJson } from "./storyFixtures/strikeThrough";
+import { linkJson } from "./storyFixtures/link";
+import { pullQuoteJson } from "./storyFixtures/pullquote";
+import { imageJson } from "./storyFixtures/image";
+import { imageBreakoutJson } from "./storyFixtures/imagebreakout";
 
-const initialValue = Value.fromJSON({
-    document: {
-      nodes: [
-        {
-          object: "block",
-          type: "paragraph",
-          nodes: [
-            {
-              object: "text",
-              leaves: [
-                {
-                  text: "A line of text in a paragraph.",
-                },
-              ],
-            },
-          ],
-        },
-      ],
-    },
-  });
+const onChange = (value: any): any => { return; };
 
-storiesOf("CivilEditor", module)
-    .add("main", () => {
-        const onChange = (value: any): any => { return; };
-        return (
-          <CivilEditor
+storiesOf("Civil Display View", module)
+    .add("paragraph", () => {
+        const initialValue = Value.fromJSON(paragraphJson);
+        return (<CivilEditor
```

Comment:
if we are going to be adding more of these I think it makes sense to build a helper method to build this;
```
const makeEditor = (initialValue: any, readOnly: boolean) => (<CivilEditor
    value={initialValue}
    onChange={onChange}
    plugins={plugins}
    readOnly={readOnly}
  />);
```